README.md

```diff
@@ -2,3 +2,32 @@ hoodie-template-app-chat
 ========================
 
 A boilerplate module you can use to start with chat in the frontend and build your own application on it.
+
+Includes the following functionality:
+
+- User Accounts
+  - sign-up
+  - sign-in
+  - sign-out
+- Send messages to the chat room
+- Recieve messages from others in the chat
+- Upload an avatar image
+- Recieve notifications when users sign-in and sign-out of the chat
+- Current user's messages are highlighted in the chat stream 
+
+![app preview](https://cloud.githubusercontent.com/assets/3051193/7662355/5acc118a-fb29-11e4-8dee-be3102d38e2d.png)
+
+How to start
+------------
+
+Create a new Hoodie app your local environment with:
```

Comment:
I think you're missing a word here,
Maybe something like "Create a new Hoodie app **in** your local environment with:" ?